neuralop/layers/spectral_convolution.py

```diff
@@ -237,61 +301,89 @@ def __init__(self, in_channels, out_channels, n_modes, incremental_n_modes=None,
 
         if output_scaling_factor is not None:
             if isinstance(output_scaling_factor, (float, int)):
-                output_scaling_factor = [[float(output_scaling_factor)]*len(self.n_modes)]*n_layers
+                output_scaling_factor = [
+                    [float(output_scaling_factor)] * len(self.n_modes)] * n_layers
             elif isinstance(output_scaling_factor[0], (float, int)):
-                output_scaling_factor = [[s]*len(self.n_modes) for s in output_scaling_factor]
+                output_scaling_factor = [
+                    [s] * len(self.n_modes) for s in output_scaling_factor]
         self.output_scaling_factor = output_scaling_factor
 
         if init_std == 'auto':
             init_std = (1 / (in_channels * out_channels))
         else:
-            init_std = 0.02
+            init_std = init_std
```

Comment:
Assumption. If we want the default to be `0.02`, then that renders the input arg `init_std` unused (if it's a `float` / not "auto").

Reply:
Good catch!